test/service/provider/administration.provider.test.js

```diff
@@ -31,70 +31,14 @@ describe('Administration provider', () => {
 			expect(administration.gemarkung).toEqual(administrationMock.district);
 		});
```

Comment:
should be corrected to:
`...
expect(administration.community).toEqual(administrationMock.gemeinde);			expect(administration.district).toEqual(administrationMock.gemarkung);
...`